vertx-web-graphql/pom.xml

```diff
@@ -19,6 +19,10 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
+  <modules>
+    <module>subscription-jwt</module>
```

Comment:
My understanding is that this module is a demo of the new feature, correct?

Then instead we should have:
- a test server in the `src/test` which will be started during the build
- a new Jest file in `tests/apollo`

There are Apollo tests already that you can inspire from. Also, the Travis setup file shows how the test server can be started and stopped.

Reply:
Right, I just added this temporarily to make sure that implementation allows for easy implementation of complete JWT authorization, I will add proper test and remove this module